### ISSUE TITLE
✅ 🐞 Switching between images in Gallery lightbox showing ugly rectangle on mobile

### ISSUE THREAD

**Issue by shimsa** [elementor/elementor#12830]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

From Elementor 3.x, switching between images in Gallery (Pro) and Basic Gallery showing ugly rectangle on mobile, see screenshot.

## Steps to reproduce

- created Gallery
- open lightbox on mobile phone
- tap to left or right arrow
- ugly rectangle showing

![gallery](https://user-images.githubusercontent.com/7617350/95989926-2e69cb80-0e2b-11eb-9a2e-6d68e7efbe17.png)

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by mcrowell** [comment]
Same here. This is the focus CSS from the theme. The code below hides it for me:
.elementor-lightbox .elementor-swiper-button:focus{outline: none;}

**Comment by shimsa** [comment]
It works, thank you. But I hope, that it will be fixed out of the box.

**Comment by lior-sh** [comment]
Hi @shimsa,

Thanks for reporting.
We will address this issue soon and release a patch.
Please follow this thread for updates.
Thanks!

**Comment by shimsa** [comment]
Same issue is with other elements, like hamburger menu and gallery functions. Can you please fix all of them?

![menu-hamburger](https://user-images.githubusercontent.com/7617350/96750671-832db900-13cc-11eb-9584-1408fcb4bf8b.png)&#160;![menu-close](https://user-images.githubusercontent.com/7617350/96750686-888b0380-13cc-11eb-9fd5-886c010e7e32.png)
![gallery](https://user-images.githubusercontent.com/7617350/96750692-8c1e8a80-13cc-11eb-8bf0-77d26d51d3b8.png)

**Comment by ReisingMedia** [comment]
Just want to chime in this post as well as it just started happening in the last couple updates and I’m relying a lot on Lightbox functionality for my current project. 

![EFEC053F-2749-4C0E-B95E-A839C43771DB](https://user-images.githubusercontent.com/71226246/97133786-17738500-1719-11eb-9773-d8666243323b.jpeg)

**Comment by nicholaszein** [comment]
Hey everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised regarding the Gallery Lightbox showing outlines on mobile devices was resolved in Elementor Core v3.27.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂 

> [!NOTE]
> If you are still experiencing issues related to SVG icon sizes on Safari, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.
